extensions/interactions/rules.json

```diff
@@ -0,0 +1,186 @@
+{
+  "CodeRepl": {
```

Comment:
Sorry, I don't understand. Do you mean create a separate copy? I don't want to do that, I want the Karma tests to check against the exact rules, so that we can be sure that the frontend service definitions match the JSON file definitions exactly.